documentation/docs/pmm-upgrade/migrating_from_pmm_2.md

```diff
@@ -13,104 +17,259 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
 
 ## Step 2: Migrate PMM 2 Server to PMM 3
 
-=== "PMM 2 with Docker volume"
-
-    Follow these manual steps to upgrade your PMM 2 Server to PMM 3:
+=== "Automated Docker migration (Recommended)"
+    Use this upgrade script for a simplified migration process:
     { .power-number}
 
-    1. Stop all PMM Server services:
+    1. Download and run the [automated migration script](https://raw.githubusercontent.com/percona/pmm/0ad7c05ae253948c779e48ff7976cb5c982af688/get-pmm.sh) to start the migration. The `-b` flag creates a backup of your PMM2 instance to ensure that your data is backed up before the migration.
```

Comment:
This URL must have changed since we captured it like this. I guess it has to be https://raw.githubusercontent.com/percona/pmm/refs/heads/v3/get-pmm.sh, TBC by @BupycHuk .

Reply:
updated with https://www.percona.com/get/pmm

Reply:
> updated with https://www.percona.com/get/pmm

I'm afraid, it still points to v2 script, while we need v3 :)